src/index.js

```diff
@@ -47,13 +52,22 @@ var page = "login";
 
         if(page === "prev") {
             const switcher = appMg.app({
-                isOnNewEv: isOnNewEv()
+                isOnNewEv: myApp.isOnNewEv()
             });
             appEl.innerHTML = switcher.view.markup;
             if(switcher.events!==undefined) {
                 switcher.events.initEvents(buildPage);
             };  
         }
+
+        //page = "details" -> for displaying a new page with candidate details
+
+        if(page === "details") {
+            const switcher = appMg.app({
+                isOnDetails: myApp.isOnDetails()
```

Comment:
I really feel that there is no real use in having multiple flags for each of our supported modules, since they should be mutually exclusive, at least at this point in time. If my assumption is correct, it would be much easier and less error prone to have a single property that uniquely identifies which of our app's modules is the current one. Food for thought. ;)